inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -151,6 +183,8 @@ def update(self) -> None:
 
         logger.debug(f"driver path = {driver_path}")
         try:
+            self.run_signature_check(driver_path)
+            
```

Comment:
**Suggestion:** Instead of using string concatenation to construct the uninstall command, use shlex.join to properly handle spaces and special characters in file paths. [security]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            import shlex
            uninstall_driver_cmd = shlex.join([CHROOT_PREFIX, '/usr/bin/rpm', '-e', '--nodeps', old_driver_name])
```